Original sputnikdao contract doesn't build

When trying to run the `build.sh` script in the `sputnikdao` directory I get a number of errors.

```
error[E0369]: binary operation `!=` cannot be applied to type `&ProposalStatus`
  --> src/lib.rs:85:22
   |
85 |         self.clone() != &ProposalStatus::Vote && self != &ProposalStatus::Delay
   |         ------------ ^^ --------------------- &ProposalStatus
   |         |
   |         &ProposalStatus
   |
   = note: an implementation of `std::cmp::PartialEq` might be missing for `&ProposalStatus`

error[E0369]: binary operation `!=` cannot be applied to type `&ProposalStatus`
  --> src/lib.rs:85:55
   |
85 |         self.clone() != &ProposalStatus::Vote && self != &ProposalStatus::Delay
   |                                                  ---- ^^ ---------------------- &ProposalStatus
   |                                                  |
   |                                                  &ProposalStatus
   |
   = note: an implementation of `std::cmp::PartialEq` might be missing for `&ProposalStatus`

…
```

--- a/sputnikdao/src/lib.rs
+++ b/sputnikdao/src/lib.rs
@@ -63,8 +63,8 @@ fn vote_requirement(policy: &[PolicyItem], num_council: u64, amount: Option<Bala
     policy[policy.len() - 1].num_votes(num_council)
 }
 
-#[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize)]
-#[cfg_attr(not(target_arch = "wasm32"), derive(Clone, Debug, Eq, PartialEq))]
+#[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, PartialEq, Debug, Clone)]
+#[cfg_attr(not(target_arch = "wasm32"), derive(Eq))]
 #[serde(crate = "near_sdk::serde")]
 pub enum ProposalStatus {
     /// Proposal is in active voting stage.

